x-pack/plugins/index_management/__jest__/components/index_table.test.js

```diff
@@ -421,7 +421,7 @@ describe('index table', () => {
     const rowIndex = 0;
     openMenuAndClickButton(rendered, rowIndex, 'forcemergeIndexMenuButton');
     snapshot(status(rendered, rowIndex));
-    expect(rendered.find('.euiModal').length).toBe(1);
+    expect(rendered.find('.euiModal').length).toBe(2);
```

Comment:
Can you explain why this value has changed?

Reply:
@alisonelizabeth I think this value changed because of how Emotion generates class names. I've updated the selector to `div.euiModal` and the test expects count to be 1 again.